release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -2318,6 +2388,16 @@ In the following tables, features are marked with the following statuses:
 |Not Available
 |Technology Preview
 
+|{cgu-operator}
```

Comment:
Should this be `cgu-operator-first`, where it has been so long since the full name was mentioned?

Reply:
yes, I agree.

Reply:
makes sense, yes.